newrelic-python.ini

```diff
@@ -32,6 +32,9 @@
 # https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/#ingest-license-key
 license_key = your_license_key
```

Comment:
Since we are providing license key using ENV so no need to add a field for license key in config file

Reply:
Commented